src/main/java/frc/robot/subsystems/leds/LEDSubsystem.java

```diff
@@ -1,22 +1,34 @@
 package frc.robot.subsystems.leds;
 
+import edu.wpi.first.math.MathUtil;
 import edu.wpi.first.wpilibj.Timer;
 import edu.wpi.first.wpilibj.util.Color;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 import static frc.robot.Constants.LEDConstants.*;
 
+import java.util.List;
+
 public class LEDSubsystem extends SubsystemBase {
     private final LEDStrip ledStrip;
 
     private final Timer blinkTimer;
     private static final double BLINK_DURATION_SECONDS = 0.5;
+    private final double APRIL_BLINK_DURATION = .2;
     private static final Color BLINK_COLOR = new Color(0, 0, 0);
     private boolean blinking = false;
+    private boolean continuous = false; //toggle in manual mode
+    public boolean manual = false; //when driver is directly controlling leds
 
     private static final double BRIGHTNESS_SCALE_FACTOR = 0.25;
+    private static final double INPUT_DEADZONE = .1;
 
     private Color color = new Color(192, 8, 254);
```

Comment:
If you're storing the colors as constants (which I think is a good idea), you might as well initialize this to `CUBE_COLOR`.